crates/bevy_window/Cargo.toml

```diff
@@ -27,6 +27,9 @@ serialize = ["serde", "bevy_ecs/serialize", "bevy_input/serialize"]
 # Enable custom cursor support
 custom_cursor = ["bevy_image", "bevy_asset"]
 
+# Enable custom icon support
+custom_window_icon = ["bevy_image", "bevy_asset"]
```

Comment:
Good: it looks like our work to enable custom cursors has untangled this web successfully.

Reply:
Yes, this was a fairly straightforward addition with the presence of `custom_cursor` as reference 🕺